grails-app/conf/application.yml

```
@@ -107,8 +107,12 @@ dataSource:
 environments:
     development:
         dataSource:
-            dbCreate: create-drop
-            url: jdbc:h2:mem:devDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
+            driverClassName: com.mysql.cj.jdbc.Driver
+            dialect: org.hibernate.dialect.MySQL8Dialect
+            dbCreate: update
+            url: jdbc:mysql://localhost:3306/miniasaas
+            username: root
+            password: Mini@saas123
```

eu aprovei, mas tem um erro aqui:
```suggestion
            password: Mini@asaas123
```

---

A senha Mini@saas123 está correta mesmo, o A foi substituído pelo @ intencionalmente.